src/controller/CHIPDevice.cpp

```diff
@@ -371,5 +371,17 @@ void Device::AddReportHandler(EndpointId endpoint, ClusterId cluster, AttributeI
     mCallbacksMgr.AddReportCallback(mDeviceId, endpoint, cluster, attribute, onReportCallback);
 }
 
+void Device::InitCommandSender()
+{
+    if (mCommandSender != nullptr)
+    {
+        mCommandSender->Shutdown();
+        mCommandSender = nullptr;
+    }
+#if CHIP_ENABLE_INTERACTION_MODEL
+    chip::app::InteractionModelEngine::GetInstance()->NewCommandSender(&mCommandSender);
```

Comment:
We should log the error if NewCommandSender return failure

Reply:
+1

Reply:
Done, thanks